### ISSUE TITLE
Error for most trinotateR functions: Error in eval(bysub, x, parent.frame()) : object 'go' not found

### ISSUE THREAD

**Issue by browngcm** [cstubben/trinotateR#2]
Hello, 
I just generated a Trinotate annotation report, and I'm trying to summarize the findings of the report with trinotateR. I can run all of the functions outlined in README.md, but when I try to summarize other aspects of the report using summary_GO, summary blast, summary_pfam, etc, I encounter this error:

> summary_GO(x)
Error in eval(bysub, x, parent.frame()) : object 'go' not found

For each of the summary functions, do I need to go through the same steps as outlined in the README.md file for the pfam summary table?

**Comment by cstubben** [comment]
Yes, you need to split the annotations into separate rows and columns first and then summarize.  This should work for GO columns and the default is `gene_ontology_blast`.

```
go <- split_go(x)
summary_GO(go)
```
